### PR TITLE
raftstore: check truncated index when calc last term

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang @disksing @zhangjinpeng1987 PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by hhkbp2** [comment]
Reset LGTM

**Comment by BusyJay** [comment]
PATL

File: src/raftstore/store/peer_storage.rs [hunk 1]
```diff
@@ -188,6 +178,24 @@ impl PeerStorage {
                 }
             };
 
+        let last_term = match raft_state.get_last_index() {
```
Comment (ngaut) [inline src/raftstore/store/peer_storage.rs]: Please abstract to a function/method

**Comment by ngaut** [comment]
PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**